blazor/diagram/layout/radial-tree.md

```diff
@@ -0,0 +1,250 @@
+        public string ReportingPerson { get; set; }
+    }
+
+    public List<RadialTreeDetails> RadialSource = new List<RadialTreeDetails>()
+    {
+        new RadialTreeDetails()   { Id = "parent", Name = "Maria Anders", Designation = "Managing Director" },
+        new RadialTreeDetails()   { Id = "1", Name = "Ana Trujillo", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   { Id = "2", Name = "Lino Rodri", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   {  Id = "3", Name = "Philip Cramer", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   {  Id = "4", Name = "Pedro Afonso", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   { Id = "15", Name = "Paul Henriot", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   { Id = "18", Name = "Laura Callahan", Designation = "Project Manager",
+            ReportingPerson = "parent" },
+        new RadialTreeDetails()   { Id = "5", Name = "Anto Moreno", Designation = "Project Lead",
+            ReportingPerson = "1" },
+        new RadialTreeDetails()   { Id = "6", Name = "Elizabeth Roel", Designation = "Project Lead",
+            ReportingPerson = "1" },
+        new RadialTreeDetails()   { Id = "7", Name = "Aria Cruz", Designation = "Project Lead",
+            ReportingPerson = "18" },
+        new RadialTreeDetails()   { Id = "8", Name = "Eduardo Roel", Designation = "Project Lead",
+            ReportingPerson = "18" },
+        new RadialTreeDetails()   { Id = "9", Name = "Howard Snyd", Designation = "Project Lead",
+            ReportingPerson = "2"  },
+        new RadialTreeDetails()   { Id = "10", Name = "Daniel Tonini", Designation = "Project Lead",
+            ReportingPerson = "2" },
+        new RadialTreeDetails()   { Id = "11", Name = "Nardo Batista", Designation = "Project Lead",
+            ReportingPerson = "3" },
+        new RadialTreeDetails()   { Id = "12", Name = "Michael Holz", Designation = "Project Lead",
+            ReportingPerson = "3" },
+        new RadialTreeDetails()   { Id = "13", Name = "Kloss Perrier", Designation = "Project Lead",
+            ReportingPerson = "4" },
+        new RadialTreeDetails()   { Id = "14", Name = "Liz Nixon", Designation = "Project Lead",
+            ReportingPerson = "4" },
+        new RadialTreeDetails()   { Id = "16", Name = "Paula Parente", Designation = "Project Lead",
+            ReportingPerson = "15" },
+        new RadialTreeDetails()   { Id = "17", Name = "Matti Kenna", Designation = "Project Lead",
+            ReportingPerson = "15" },
+    };
+}
+```
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Layout/)
```

Comment:
You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Layout/).